assets/scss/layout/_more.scss

```diff
@@ -4,29 +4,18 @@
 [data-layout='more'] {
   @include above('page-break') {
     display: grid;
-    grid-template-columns:
-      [three-start] 1fr
-      [two-start] calc(var(--margin) * 3)
-      [three-end] 1fr
-      [one-start] calc(var(--margin) * 3)
-      [two-end] 1fr
-      [one-end];
-    grid-template-rows:
-      [one-start] auto
-      [two-start] size('shim', 'px')
-      [one-end] auto
-      [three-start] size('shim', 'px')
-      [two-end] auto
-      [three-end];
+    grid-template-columns: 1fr 1fr;
```

Comment:
Since we have a max-width on it, this grid could probably use `auto-fit` instead of a media-query. Not that it really makes a difference in this case…